.github/workflows/main.yml

```
@@ -9,75 +9,63 @@ on:
     branches:
       - '**'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  build_and_test:
+  build_core:
+    name: Build Stencil
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+      - uses: ./.github/workflows/actions/build-core
+
+  format:
+    name: Format Check
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+      - uses: ./.github/workflows/actions/format
+
+  unit_test:
+    needs: [build_core]
     strategy:
+      fail-fast: false
       matrix:
-        node: ['12', '14', '16']
-        os: ['ubuntu-latest', 'windows-latest']
+        node: [ '12', '14', '16' ]
+        os: [ 'ubuntu-latest', 'windows-latest' ]
```

Prettier uses no bracket spacing in this case:
```suggestion
        node: ['12', '14', '16']
        os: ['ubuntu-latest', 'windows-latest']
```

---

Note to self: I should get prettier running in this dir (on YAML)

---

Fixed in https://github.com/ionic-team/stencil/pull/3332/commits/25ee864ff645ab91b9f4a7fabab1178ddeee1a26